admin/class-automation.php

```diff
@@ -170,6 +178,26 @@ public static function filter__apple_news_exporter_slug( $slug, $post_id ) {
 		return $slug;
 	}
 
+	/**
+	 * A callback function for the apple_news_exporter_title filter.
+	 *
+	 * @param string $title   The title to use.
+	 * @param int    $post_id The post ID associated with the title.
+	 *
+	 * @return string The filtered title value.
+	 */
+	public static function filter__apple_news_exporter_title( $title, $post_id ) {
+		self::$original_title = $title;
+		$rules                = self::get_automation_for_post( $post_id );
+		foreach ( $rules as $rule ) {
+			if ( 'headline.prepend' === ( $rule['field'] ?? '' ) ) {
+				$title = sprintf( '%s %s', $rule['value'] ?? '', self::$original_title );
```

Comment:
Unconditionally applying the `sprintf()` here in conjunction with the null coalescing to `''` reads a little strangely to me, since it implies that the user would want there to be leading spaces in the title in the event that `value` was missing. I wonder whether there should be a string length check prior to changing `$title`?